e2e/test/netstandard20/ProvisioningE2ETests.cs

```diff
@@ -1012,7 +1105,22 @@ private async Task ConfirmRegisteredDeviceWorks(DeviceRegistrationResult result,
             }
         }
 
-        private async Task<SecurityProvider> CreateSecurityProviderFromName(AttestationType attestationType, EnrollmentType? enrollmentType, string groupId, ReprovisionPolicy reprovisionPolicy, AllocationPolicy allocationPolicy, CustomAllocationDefinition customAllocationDefinition, ICollection<string> iothubs)
+        private async Task ConfirmExpectedDeviceCapabilities(DeviceRegistrationResult result, DeviceCapabilities capabilities)
```

Comment:
I think this isn't testing the right thing. In my mind, the right scenario would be:
1. DPS Service sets the capabilities (via service client)
2. Device performs DPS provisioning.
3. Device connects to IoT Hub and queries the Twin to see the capabilities.

The test below is querying RegistryManager for some reason - that isn't a device-side API.

Reply:
I'll change this to use the device side twin to verify the flag is set as expected instead of the registry manager, sure

Reply:
Actually, I don't see a way to query this information from a device side query. We have to use the registry manager here to see if the device is an edge device or not.